routes/products.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const productsRouter = express.Router();
+
+const {createProduct, getProductById, getAllProducts} = require('../db');
+
+productsRouter.get('/', async (req, res, next) => {
+    try {
+        const products = await getAllProducts();
+
+        res.send(products)
+
+    } catch (error) {
+        next(error)
+    }
+})
+
+productsRouter.get('/:productId', async (req, res, next) => {
+    try {
+        const product = await getProductById(req.params.productId);
```

Comment:
Maybe check if the param is a number (or can be coerced to a number) before throwing it into the function.  This could solve potential errors.  Maybe send a 404 if there is no matching product.